### PR TITLE
FA_DNA presets updated

### PR DESCRIPTION
**Pull request by Arslan-Siraj**
## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes. (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out 
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. If you click in the column that lists the failed tests you will get detailed error messages.
- Or click on the action: e.g., for clang-format linting

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h [hunk 1]
```diff
@@ -309,7 +309,12 @@ namespace OpenMS
     "A:C2",
     "A:C2-HPO3",
     "A:C2-H3PO4",
-    "A:C2-H2O"
+    "A:C2-H2O",
+
+    "d:",
```
[the remark this answers is not in the document]
Comment (Arslan-Siraj) [reply src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h:314]: she said might be find adduct of d, without C1 addition, but we can confirm when alex come back from vacations.

[the remark this answers is not in the document]
Comment (timosachsenberg) [reply src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h:314]: You can try and see if you find some of them.

File: src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h [hunk 2]
```diff
@@ -1102,7 +1102,7 @@ namespace OpenMS
     const String RNA_UCGA = "UCGA";
     const String DNA_TCGAd = "TCGAd";
     const String RNA_CGA = "CGA";
-    const String DNA_CGA = "CGA";
+    const String DNA_CGA = "CGAd";
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h:1105]: DNA_CGAd

**Review by timosachsenberg — CHANGES_REQUESTED** [review]
Variable name needs to be fixed - otherwise looks good

**Pull request merged**